tests/test_lex.py

```diff
@@ -63,6 +63,17 @@ def test_unbalanced_exception():
         pass
 
 
+def test_lex_single_quote_err():
+    "Ensure tokenizing \"' \" throws a LexException that can be stringified"
+    # https://github.com/hylang/hy/issues/1252
+    try:
+        tokenize("' ")
+    except LexException as e:
+        assert str(e)
```

Comment:
The string is currently `"\x1b[33m\x1b[22mLexException: Could not identify the next token.\n\n\x1b[39m\x1b[22m"`. If we check for equality against that, we're testing the use of ANSI escape codes, which doesn't seem right.